nautobot/extras/models/groups.py

```diff
@@ -256,12 +271,12 @@ def get_queryset(self):
     @property
     def members(self):
         """Return the member objects for this group."""
-        return self.get_queryset()
+        return self.get_group_queryset()
```

Comment:
What's the difference now between `DynamicGroup.get_queryset()` and `DynamicGroup.get_group_queryset()`? Is using `get_queryset()` now deprecated, or does it not work as desired any more? Especially wondering in terms of backwards compatibility with 1.3 DynamicGroups ORM API.

Reply:
`get_queryset()` will continue to behave as it did, giving the base queryset for a group based on its filter (if any). It has not been modified. So `get_group_queryset()` accounts for the base filter AND the filters of the child groups. 

This isn't quite finished as some of the assertions about either filter OR child groups BUT NOT BOTH has not yet been enforced nor has it been tested.